product_multi_ean/views/product_view.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="product_normal_form_view_multi_ean13" model="ir.ui.view">
+            <field name="name">product.normal.form.multi.ean13</field>
+            <field name="model">product.product</field>
+            <field name="inherit_id" ref="product.product_normal_form_view" />
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='ean13']" position="attributes">
+                    <attribute name="readonly">True</attribute>
+                </xpath>
+                <xpath expr="//field[@name='ean13']" position="after">
+                    <field name="ean13_ids">
+                        <tree string="EAN13" editable="bottom">
+                            <field name="sequence" widget="handle"/>
+                            <field name="name"/>
+                        </tree>
+                    </field>
+                </xpath>
+            </field>
+        </record>
+
+
+        <record id="product_search_form_view_multi_ean13" model="ir.ui.view">
+            <field name="name">product.product.search.multi.ean13</field>
+            <field name="model">product.product</field>
+            <field name="inherit_id" ref="product.product_search_form_view"/>
+            <field name="arch" type="xml">
+                <field name="name" position="after">
+                   <field name="ean13"/>
```

Comment:
The resulting label ``Search Main EAN13 for:`` in the search suggestions might be confusing, since the overridden ``search()`` method of the model actually searches the ``ean13_ids``.
Maybe this should read ``<field name="ean13" string="EAN13"/>``